server.js

```diff
@@ -25,11 +27,13 @@ Sentry.init({
   },
 })
 
-if (+process.argv[2]) {
-  config.public.bind.port = +process.argv[2]
-}
-if (process.argv[3]) {
-  config.public.bind.address = process.argv[3]
+if (process.env.NODE_CONFIG_ENV !== 'test') {
```

Comment:
Switching from a dynamic `await import` to a module-level import also meant that this was picking up the test runner args here https://github.com/badges/shields/blob/9447077c0836bfe5b4f76f534ebf98a0f402763f/.github/actions/core-tests/action.yml#L13 under test.